pkg/k8sops/generators_test.go

```diff
@@ -231,6 +231,7 @@ func TestGenerateStatefulSet(t *testing.T) {
 					},
 				},
 			},
+			VolumeClaimTemplates: clusterSpec.PersistentVolumeClaims,
```

Comment:
Can you add a claim template to the fixture cluster and validate that it shows up as expected here?

Reply:
Oh I did that but I never committed it 😅 oops